test/SILOptimizer/definite_init_actor.swift

```diff
@@ -283,6 +355,11 @@ actor TaskMaster {
 
     ////// for the initializer
     // CHECK-LABEL: @$s4test10TaskMasterCACyYacfc : $@convention(method) @async (@sil_isolated @owned TaskMaster) -> @owned TaskMaster {
+    //   Initial hop to the generic executor.
+    //   TODO: This should be easy to remove.
```

Comment:
Is there any situation when we can't remove this initial hop...?

Reply:
Yes, absolutely. It's like any other hop after a bit of inlining: there's a hop to the generic executor on entry, and another hop later in the function, and we have to honor that initial hop if there's work done within that duration that might depend on it.